Month1/Week2/Section1/Exercise4/Main4.cpp

```diff
@@ -23,31 +23,28 @@ o
 */
 #include<iostream>
 #include<string>
-class Printer
+class FPrinter
 {
 private:
 	std::string OutputString;
 public:
-	Printer()
+	FPrinter()
 	{
 	}
-	Printer(std::string OutputString)
-	{
-		OutputString = "";
-	}
+	
 	void InputString() 
 	{
-		std::string Outputstring;
+		std::string EntryString;
 		std::cout << "Welcome! Please enter a string:" << std::endl;
-		std::getline( std::cin,Outputstring);
+		std::getline( std::cin,EntryString);
```

Comment:
Spaces are not following convention here.

Reply:
Changed